css/style.css

```diff
@@ -1 +1,8 @@
-/* file for your css */
\ No newline at end of file
+/* file for your css */
+#arrow-left {
+  margin-left: 0 !important;
+}
+
+#arrow-right {
+  margin-left: 0 !important;
+}
```

Comment:
Here is my CSS for red and green on win or fail:

    .has-success {
        background-color: green;
    }

    .has-fail {
        background-color: red;
    }